content/blog/2021/06/2020-06-03-digester-removal.adoc

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "commons-digester library removal (breaking changes!)"
+tags:
+- jenkins
+- core
+- developer
+author: batmat
+description: "The commons-digester:2.1 library has been removed from Jenkins Core. A few plugins will be broken with the upcoming release."
+---
+
+Keeping up with our goal to clean up some of the technical debt inside Jenkins Core, we are happy to report we were able to remove a long-deprecated library.
+
+link:https://github.com/jenkinsci/jenkins/pull/5320[Jenkins does not depend anymore] on link:https://search.maven.org/artifact/commons-digester/commons-digester/2.1/jar[commons-digester:2.1], a version that has been released in 2010.
+
+Here is the list of plugins that did _not_ have their fix released at the time of writing.
+They will start to break with the weekly on the 7th of June, expected to be the 2.297.
+
+* link:https://plugins.jenkins.io/teamconcert[teamconcert]
+* link:https://plugins.jenkins.io/vs-code-metrics[vs-code-metrics]
+* link:https://plugins.jenkins.io/BlameSubversion[BlameSubversion]
+* link:https://plugins.jenkins.io/javatest-report[javatest-report]
+* link:https://plugins.jenkins.io/vss[vss]
+* link:https://plugins.jenkins.io/synergy[synergy]
+* link:https://plugins.jenkins.io/config-rotator[config-rotator]
+* link:https://plugins.jenkins.io/harvest[harvest]
+* link:https://plugins.jenkins.io/cmvc[cmvc]
+
+[NOTE]
+====
+It is always a good idea to update all your plugins _before_ upgrading Jenkins core.
+====
+
+Please reach out on the link:/mailing-lists/[developers’ list] with any questions or suggestions.
+
+=== Resources
+
+link:https://github.com/jenkinsci/jenkins/pull/5320[PR-5320]:: removing `commons-digester:2.1`.
```

Comment:
a screenshot of the post in the description would be good to include to make sure all this renders correctly

Reply:
Care to elaborate? I'm confused by this request.

Reply:
normally someone would take a full page screenshot of a post and put it in the description so others don't have to check it out locally.

Reply:
oh ok, I generally don't do that for simple entries with text, sticking with GH renderer. Looking into this then.